lib/object-path-immutable-transaction.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+const _assign = require('lodash.assign');
+const objectPath = require('object-path');
+const _isPlainObject = require('lodash.isplainobject');
+
+function normalizePath(path) {
+	return Array.isArray(path) ? path : ('' + path).split('.');
```

Comment:
My thinking was to make sure that the call to `split` would always work. I dunno, might be the wrong call.